apps/frontend/src/renderer/components/task-detail/task-review/ConflictResolver/ResolutionControls.tsx

```
@@ -0,0 +1,151 @@
+        <div className="flex items-center gap-2">
+          <span className="text-xs text-muted-foreground">Resolution mode:</span>
+          <div className="flex rounded-md border border-border overflow-hidden">
+            <button
+              onClick={() => onResolutionModeChange('file')}
+              className={cn(
+                "px-3 py-1 text-xs transition-colors",
+                resolutionMode === 'file'
+                  ? "bg-primary text-primary-foreground"
+                  : "bg-background hover:bg-muted"
+              )}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)